templates/src/python/_parallel_compile_patch.include

```diff
@@ -0,0 +1,65 @@
+"""Patches the compile() to allow enable parallel compilation of C/C++.
```

Comment:
Nit: Maybe change the filename to `_parallel_compile_patch.py.include` to indicate this is a Python file.